tests/e2e-ui-tests/page-objects/mobile/workflow/main.po.js

```diff
@@ -46,21 +46,21 @@ var WorkflowPage = function() {
     },
     dragSlider: function(percentage) {
       var value = null;
-      switch(percentage) {
-        case percentage < 13:
-          value = 0;
-          break;
-        case percentage < 38:
-          value = 25;
-          break;
-        case percentage < 63:
-          value = 50;
-          break;
-        case percentage < 88:
-          value = 75;
-          break;
-        default:
-          value = 100;
+      switch (percentage) {
```

Comment:
This is example of the code taht we should not test. 
That's example step and we can continue that without putting any values there - they assume to be zero. 
We can even improve example step to make some values preselected. 

We should move away with complexity of the steps to really test workflow code.

Reply:
There is some legacy code, which is not needed, but in tests it is not used (i.e. example steps are not being tested)